core-cms/settings_custom.py

```diff
@@ -0,0 +1,297 @@
+# TACC WMA SAD CMS SITE: DEV.CEP.TACC.UTEXAS.EDU
+# CUSTOM SETTINGS VALUES.
+
+########################
+# DJANGO SETTINGS
+########################
+
+DEBUG = True       # False for Prod.
+
+# Specify allowed hosts or use an asterisk to allow any host and simplify the config.
+# ALLOWED_HOSTS = ['hostname.tacc.utexas.edu', 'host.ip.v4.address', '0.0.0.0', 'localhost', '127.0.0.1']   # In production.
+ALLOWED_HOSTS = ['dev.cep.tacc.utexas.edu']
+
+# Boolean check to see if ldap is being used by the site.
+# Requires django-auth-ldap ≥ 2.0.0
+LDAP_ENABLED = True
+
+########################
+# DJANGO CMS SETTINGS
+########################
+
+# CMS Site (allows for multiple sites on a single CMS)
+SITE_ID = 1
+CMS_TEMPLATES = (
+    # Customize this
+    # FAQ: First template is default template
+    # REF: http://docs.django-cms.org/en/latest/how_to/install.html#templates
+    ('example-cms/templates/fullwidth.html', 'Fullwidth'),
+    # Support standard template for demo purposes
+    ('fullwidth.html', 'Standard Fullwidth'),
+
+    # Support Portal pages for demo purposes
+    ('home_portal.html', 'Standard Portal Homepage'),
+    ('guide.html', 'Guide'),
+    ('guides/getting_started.html', 'Guide: Getting Started'),
+    ('guides/data_transfer.html', 'Guide: Data Transfer'),
+    ('guides/data_transfer.globus.html', 'Guide: Globus Data Transfer'),
+    ('guides/portal_technology.html', 'Guide: Portal Technology Stack'),
+    ('style_guide.html', 'Style Guide')
+)
+
+CEP_AUTH_VERIFICATION_ENDPOINT='https://dev.cep.tacc.utexas.edu'
```

Comment:
@xandroc @taoteg,

1. Would this `CEP_AUTH_VERIFICATION_ENDPOINT` be different for each server?
2. If so, then would it be for `secrets.py`, not `settings_custom.py`, thus not need to be here?

Reply:
Ahh! Great catch, Wes, yes. We should relocate that value into the secrets.py.

This value should reflect the same value as the first entry in `ALLOWED_HOSTS` for any given machine on the TACC network (`dev|pprd|prod.project_domain.tacc.utexas.edu`).

Reply:
Ideally we'll only have secrets in `secrets.py`, we'll use `settings_custom.py` to track custom settings across environments, so each env will have and commit its own `settings_custom.py`. Camino will load these in a similar way as to how different image tags are loaded on each environment

Reply:
Yes, that is the long term plan once we have the CMS settings moved into the Deployments repo and mirroring portals. 

That said, it will have to be done in a  follow on task/ticket in a subsequent sprint. There are just too many portals currently being deployed (over the next two week) for me to be remotely comfortable with additional to changes the deployment setup at this time having just gotten everything working cleanly as of Friday (and with there still being several portals we need to roll out the changes in order to complete aligning our live sites).

Reply:
Thank you, both. All makes sense.

Should not be secret value, but must be until we support per-env settings, which is not immediate.

Based on that understanding, I will **remove** the `CEP_AUTH_VERIFICATION_ENDPOINT` from:
- #74

Reply:
Ah, I never added it to the other sites. So, I don't need to remove it from them. 😅

Reply:
Ha! Perfect. Okay, I am updating all the hosts to include the value in secrets.py.
we should be good to merge this after we pickup the UTRC stuff in #80 and the other portal settings in #74.
main <-- #77 <-- #74 <-- #80.
Make it so!
=)